test/eChronicler/e_chronicler/journal_entry_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule EChronicler.EChronicler.JournalEntryTest  do
+  use EChronicler.DataCase, async: true
+  import Ecto.Query, warn: false
+
+  alias EChronicler.EChronicler.JournalEntries
+  alias EChronicler.EChronicler.JournalEntry
+
+  test "format_datetime/1 returns formatted string if correct" do
+    correct_datetime = %EChronicler.EChronicler.JournalEntry{title: "Awesome Blog Post", author: "Grace", entry: "Test.", inserted_at: ~U[2021-12-16 16:36:15.549610Z]}
```

Comment:
I think because you are importing this:
`alias EChronicler.EChronicler.JournalEntry`

...then you don't have to use the full module name chain here:
`%EChronicler.EChronicler.JournalEntry{...`

Try just 
`%JournalEntry{...`
... and see if it works 🤷